.gitignore

```diff
@@ -3,6 +3,7 @@ node_modules
 .idea
 docs/.vuepress/dist
 yarn-error.log
+yarn.lock
```

Comment:
Bit of a drive by review but glanced and saw this, should we be ignoring the yarn.lock file?

Reply:
I don't know...
I just run `make install && make build` and found this file. I mean it's a generated file from compile, maybe we could  ignore it?

Reply:
Usually a `yarn.lock` is commited to pin dependencies, but sounds like here that has never been done. I don't know enough about these build/install scripts to know if there is a reason for this, so same as you! 😅 😆 🙌 

Umm, wow it does `npm install` then `yarn install`, ideally we should probably take look at this. Lemme just check with folks but at the least I think we shouldn't ignore the lock file just yet.

Reply:
Ok yeah so we _are_ pinning our dependencies here (if I make a change to the lock file git tells me).

We definitely shouldn't gitignore it.

Reply:
OK, got it.

Reply:
> Umm, wow it does `npm install` then `yarn install`

😬

Reply:
Yeah, this should be changed:

```diff
- 	npm install -g netlify-cli@16.5.1
+ 	yarn global add netlify-cli@16.5.1 
```